servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/OutlierDetectorConfig.java

```diff
@@ -26,10 +26,10 @@
  * XDS outlier detector configuration.
  * <p>
  * See the <a href="https://www.envoyproxy.io/docs/envoy/v1.29.0/api-v3/config/cluster/v3/
- * outlier_detection.proto#envoy-v3-api-msg-config-cluster-v3-outlierdetection"> Envoy docs</a> for the official
+outlier_detection.proto#envoy-v3-api-msg-config-cluster-v3-outlierdetection"> Envoy docs</a> for the official
```

Comment:
I can't figure out how to break this line and still be a valid url for the javadoc. Any suggestions?

Reply:
You can add `gradle/checkstyle/suppressions.xml` file under this module to suppress this one.

Consider removing an extra space: `> Envoy docs</a>`

Reply:
Thank you, done.